tracing/tracing-test.sh

```diff
@@ -26,6 +28,13 @@ jaeger_docker_container_name="jaeger"
 cleanup(){
 	stop_jaeger 2>/dev/null || true
 	.ci/configure_tracing_for_kata.sh disable
+
+	# The tests worked so remove the logs
+	if sudo [ -d "$TRACE_LOG_DIR" ] && [ "$TRACE_LOG_DIR" != "/" ]
+	then
+		info "Removing cached trace logs from $TRACE_LOG_DIR"
+		sudo rm -rf "$TRACE_LOG_DIR"
```

Comment:
@jodh-intel  I am  not clear when the trace logs are keep , seems that they are removed all the time ?

Reply:
Good catch @jcvenegas!! I've moved the deletion logic to "main" now.